resources/views/application/application_dashboard.php

```diff
@@ -28,11 +27,18 @@
                     </div>
 
                     <div class="application-actions">
+
+                        <?php if ($current_status === 'Aceptado') : ?>
+                                <a href="/apply/requireddocuments" class="" style="margin-right: 1rem;">
+                                    <i class="fas fa-upload"></i> Subir documentos
```

Comment:
Debería ser más descriptivo

```suggestion
                                    <i class="fas fa-upload"></i> Subir documentos finales
```